modules/persistent-storage-local-install.adoc

```diff
@@ -34,6 +34,27 @@ To block the Local Storage Operator from inheriting the cluster-wide default sel
 $ oc annotate project openshift-local-storage openshift.io/node-selector=''
 ----
 
+. Optional: Allow local-storage operator installation on single node clusters.
++
+To allow installation of local-storage operator in single node clusters, we need to add following annotation to the namespace in which operator is being installed.
```

Comment:
Change to: To allow installation of local storage operator in single node clusters, add the following annotation to the namespace in which operator is being installed: